rules.d/50-udev-default.rules.in

```diff
@@ -62,6 +62,8 @@ SUBSYSTEM=="accel", GROUP="render", MODE="{{GROUP_RENDER_MODE}}"
 SUBSYSTEM=="misc", KERNEL=="sgx_enclave", GROUP="sgx", MODE="0660"
 SUBSYSTEM=="misc", KERNEL=="sgx_vepc", GROUP="sgx", MODE="0660"
 
+SUBSYSTEM=="misc", KERNEL=="ntsync", GROUP="ntsync", MODE="0666"
```

Comment:
0666 ??? Hmm, then why the group needs to be introduced??
I think, as usual, 0660 should be natural. Please elaborate more otherwise.

Reply:
Appoplogies I thought both was required, I'll remove the group

Reply:
Much simpler, thanks